src/Gui/NaviCube.cpp

```diff
@@ -704,8 +706,8 @@ void NaviCubeImplementation::prepare()
     if (m_PickingFramebuffer)
         delete m_PickingFramebuffer;
     m_PickingFramebuffer =
-        new QtGLFramebufferObject(2 * m_CubeWidgetSize, 2 * m_CubeWidgetSize,
-                                  QtGLFramebufferObject::CombinedDepthStencil);
+        new QOpenGLFramebufferObject(2 * m_CubeWidgetSize, 2 * m_CubeWidgetSize,
+                                  QOpenGLFramebufferObject::CombinedDepthStencil);
```

Comment:
I'm surprised pre-commit didn't fix this alignment 🤔
```suggestion
                                     QOpenGLFramebufferObject::CombinedDepthStencil);
```